indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -331,7 +328,18 @@ public static boolean runJobs(List<Jobby> jobs, HadoopDruidIndexerConfig config)
         Path workingPath = config.makeIntermediatePath();
         log.info("Deleting path[%s]", workingPath);
         try {
-          workingPath.getFileSystem(injectSystemProperties(new Configuration())).delete(workingPath, true);
+          if( workingPath.getFileSystem(injectSystemProperties(new Configuration())).delete(workingPath, true) ){
```

Comment:
There are two style guides in the druid repo, one for IntelliJ and one for eclipse , do you mind applying those styles here?